x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -665,6 +649,37 @@ export async function getInstallation(options: {
   return savedObject?.attributes;
 }
 
+/**
+ * Return an installed package with his related assets
+ */
+export async function getInstalledPackageWithAssets(options: {
+  savedObjectsClient: SavedObjectsClientContract;
+  pkgName: string;
+  logger?: Logger;
+}) {
+  const installation = await getInstallation(options);
+  if (!installation) {
+    return;
```

Comment:
What happens in the `getPackageAssetsMap` function in these case when this function just returns? It seems that it would go [here](https://github.com/elastic/kibana/pull/173965/files#diff-baf8294e3f474e4f2436b376ab8b1c1c8a611c0d0455ff88e78563c613deeb4dR725-R727) but `installedPackageWithAssets` would be undefined? 
Just checking that we don't have unhandled exceptions.

Reply:
yes installedPackageWithAssets will return `undefined`  and `getPackageAssetsMap` will fallback to the registry, this the codepath used by the `configs` tab for non installed package (I tested that tab it seems to works well)